analyzer-models/src/test/java/ffb/test/utilities/GenericTestUtils.java

```diff
@@ -0,0 +1,76 @@
+package ffb.test.utilities;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.junit.Assert;
+
+public abstract class GenericTestUtils {
+    private static final List<String> GETTER_METHOD_PATTERNS = List.of("^get[A-Z].*", "^is[A-Z].*");
+
+    public static void validateGetMethodsReturnNonNullValue(List<?> testObjects) throws InvocationTargetException, IllegalAccessException {
+        List<Method> getterMethods = new ArrayList<Method>();
+
+        getterMethods = Stream.of(testObjects.get(0).getClass().getMethods())
+                .filter(GenericTestUtils::isGetterMethod)
+                .collect(Collectors.toList());
+
+        for (var testObj : testObjects) {
+            for (Method method : getterMethods) {
+                Assert.assertNotNull(method.invoke(testObj));
+            }
+        }
+    }
+
+    public static void validateGetMethodsReturnNonNullValue(Object testObject) throws InvocationTargetException, IllegalAccessException {
+        Stream.of(testObject.getClass().getMethods())
+                .filter(GenericTestUtils::isGetterMethod)
+                .forEach(method -> {
+                            try {
+                                method.invoke(testObject);
+                            } catch(Exception e) {
+                                Assert.fail("Exception: " + e);
+                            };
+                });
+
+        for (Method method : testObject.getClass().getMethods()) {
```

Comment:
This block can be moved into your `forEach` loop. It's effectively duplicate code. Issue #19.